packages/remix-server-runtime/cookies.ts

```diff
@@ -57,19 +57,19 @@ export interface Cookie {
   parse(
     cookieHeader: string | null,
     options?: CookieParseOptions
-  ): Promise<any>;
+  ): Promise<Type | null>;
 
   /**
    * Serializes the given value to a string and returns the `Set-Cookie`
    * header.
    */
-  serialize(value: any, options?: CookieSerializeOptions): Promise<string>;
+  serialize(value: Type, options?: CookieSerializeOptions): Promise<string>;
```

Comment:
Probably a good idea to rename `Type` to `Value`

```suggestion
  serialize(value: Value, options?: CookieSerializeOptions): Promise<string>;
```